common/org.eclipse.tracecompass.incubator.releng-site/category.xml

```diff
@@ -89,6 +89,9 @@
    <bundle id="org.eclipse.wst.jsdt.manipulation"/>
    <bundle id="org.eclipse.nebula.widgets.opal.commons"/>
    <bundle id="org.eclipse.nebula.widgets.opal.duallist"/>
+   <bundle id="jakarta.activation-api" version="1.2.2"/>
+   <bundle id="jakarta.xml.bind-api" version="2.3.3"/>
+   <bundle id="javax.inject"/>
```

Comment:
I worry if this should be jakarta.inject.jakarta.inject-api in order to get the same version than in orbit-aggregation?

Reply:
I'll try that.

Reply:
I'm not able to make it work with this one. Maybe it's ok to use as-is. Soon we have the 10.0 version.

Reply:
For some reason plug-in `org.eclipse.ease.modules.platform` needs exact version `1.0.0` of `javax.inject`. Here that's what we get from the MANIFEST.mf:
```
`javax.inject;bundle-version="1.0.0"`
```